README.md

```diff
@@ -41,13 +41,20 @@ If you'd like to also classify nodes in ServiceNow's CMDB, then the module:
 **Note:** If you're interested in both the ServiceNow CMDB external data and node classification features of this module, then you can just read the [classification](#ServiceNow-node-classification) section since that depends on the CMDB external data feature.
 
 #### ServiceNow CMDB as trusted external data
-Apply the `servicenow_cmdb_integration` class on all of your compilers. This looks something like:
+Apply the `servicenow_cmdb_integration` class on all of your compilers. You may include a user and password or an oauth token This looks something like:
```

Comment:
Eh, I plan on updating this after `Getting Started` doc so we can leave out mention of oauth token for now in user-facing part of README.